### PR TITLE
Feature/20230327 julian

### PR DESCRIPTION
**Pull request by Julian0701**
### DEVELOP
- [x] 500.tsx
- [x] 404.tsx
- [x] coming-soon.tsx
- [x] deposit_history_modal.tsx
- [x] withdrawal_history_modal.tsx

### CHECK LIST

- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component: 3
   - src/pages/coming-soon.tsx
   - src/components/deposit_history_modal/deposit_history_modal.tsx
   - src/interfaces/tidebit_defi_background/display_accepted_deposit_order.ts
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 0

### UML

- [Sequence Diagram 001 (interaction between user, frontend and backend)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00001.md)
- [Sequence Diagram 002 (background mechanism)](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDSD00002.md)
- [Component Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCP00001.md)
- [Class Diagram](https://github.com/CAFECA-IO/Documents/blob/main/TBD/TBDCD00001.md)
- [Global context document](https://www.notion.so/Global-Context-public-d93ac7c941c94fb6b173d34a7bc4e959)
- [Table of interface](https://github.com/CAFECA-IO/EventReport/blob/main/meeting/20230106_tbd_mvp.md)

### CODE REVIEW

**Review by Julian0701 — COMMENTED** [review]
fix these issues

File: src/components/cta/cta.tsx [hunk 1]
```diff
@@ -56,27 +55,25 @@ const Cta = () => {
...
-            <div className="mx-auto pt-2 md:pt-8">
-              <a
-                href="/trading"
-                className="px-2 py-1 font-normal xxs:text-xxs sm:py-2 sm:px-5 sm:text-base xl:py-3 xl:px-5 xl:text-lg"
+          </div>
+          <div className="mx-5 mt-4 max-w-md text-sm font-normal leading-7 tracking-0.02rem text-white md:text-xl">
+            {t('HOME_PAGE.CTA_DESCRIPTION')}
+          </div>
+          <div className="mx-auto pt-4 md:pt-8">
+            <Link href="/trade/cfd/ethusdt">
```
Comment (Julian0701) [inline src/components/cta/cta.tsx:69]: constants/api_request.ts

File: src/components/reserve_ratio/reserve_ratio.tsx [hunk 2]
```diff
@@ -72,12 +72,7 @@ const ReserveRatio = () => {
               >
                 <path d="M22 12h-4l-3 9L9 3l-3 9H2"></path>
               </svg> */}
-                    <Image
-                      src="/elements/tether-seeklogo.com.svg"
-                      width={50}
-                      height={50}
-                      alt="tether-seeklogo.com.svg"
-                    />
+                    <Image src="/asset_icon/usdt.svg" width={50} height={50} alt="USDT_icon" />
```
Comment (Julian0701) [inline src/components/reserve_ratio/reserve_ratio.tsx:75]: get asset by marketContext

File: src/components/trial_component/trial_component.tsx [hunk 3]
```diff
@@ -63,12 +63,7 @@ const TrialComponent = () => {
               >
                 <path d="M22 12h-4l-3 9L9 3l-3 9H2"></path>
               </svg> */}
-                <Image
-                  src="/elements/tether-seeklogo.com.svg"
-                  width={50}
-                  height={50}
-                  alt="tether-seeklogo.com.svg"
-                />
+                <Image src="/asset_icon/usdt.svg" width={50} height={50} alt="USDT_icon" />
```
Comment (Julian0701) [inline src/components/trial_component/trial_component.tsx:66]: get data from marketContext

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**